common/tools.py

```diff
@@ -77,3 +77,23 @@ def setup_data_folder(data_folder):
     """
 
     Path(data_folder).mkdir(parents=True, exist_ok=True)
+
+
+def intify(obj):
+    """Takes an object that is a recursive composition of primitive types, lists and dictionaries, and returns an
```

Comment:
Some nits:

Formatting within docs needs double quotes.

Also, given the docs are not a oneliner, break after """ (for both opening and close):
```
"""
Docs
"""

code
```

Reply:
Fixed in 8909d91744bc8298f91aaca2eee48db72ee5e3d4, also in several other files.